.travis.yml

```diff
@@ -1,5 +1,10 @@
+sudo: required
```

Comment:
I'm not sure. travis-ci docs tell you to add it but do not invoke sudo directly (except in the docker-compose example at the very end). Might be a peculiarity of how they do builds.

https://docs.travis-ci.com/user/docker/#Branch-Based-Registry-Pushes